syntax = "proto3";

package test.integration.filters;

message LoggingTestFilterConfig {
  string logging_id = 1;
  string upstream_cluster_name = 2;
}
